src/OmniSharp/project.json

```diff
@@ -4,7 +4,9 @@
         "warningsAsErrors": true
     },
     "dependencies": {
+        "OmniSharp.Abstractions": "1.0.0-*",
         "OmniSharp.Stdio": "1.0.0-*",
+        "OmniSharp.Roslyn.CSharp": "1.0.0-*",
```

Comment:
This is just temporary, to force a reference to the assembly, for true plugins we'd have a set of folders to scan for assemblies.